src/pages/Tailwind/Pool/index.tsx

```diff
@@ -1,52 +1,54 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import PageWrapper from 'components/Tailwind/Layout/PageWrapper'
 import PageHeaderLeft from 'components/Tailwind/Layout/PageHeaderLeft'
 import PageHeaderRight from './PageHeaderRight'
 import PoolColumns from './PoolColumns'
 import ExpandablePoolRow from './ExpandablePoolRow'
-import { PoolData } from './models/PoolData'
-import { ChainId } from '@sushiswap/sdk'
-import { HALO, USDT, XSGD, USDC } from '../../../constants'
+import { LIQUIDITY_POOLS_ADDRESSES } from 'constants/pools'
+import { useLPTokenAddresses } from 'halo-hooks/useRewards'
+import { updatePools } from 'state/pool/actions'
+import { CachedPool } from 'state/pool/reducer'
+import { useDispatch } from 'react-redux'
+import { AppDispatch } from 'state'
```

Comment:
should these be like this to be consistent casing?

```suggestion
import { cachedPool } from 'state/pool/reducer'
import { useDispatch } from 'react-redux'
import { appDispatch } from 'state'
```

Reply:
`CachedPool` & `AppDispatch` are data types so I would keep its casing.